assembl/static2/css/common.scss

```diff
@@ -381,7 +381,7 @@ strong {
 
 @media screen and (max-width: $screen-xxs-max) {
   .content-section {
-    padding: $margin-section-xs 0;
+    padding: $margin-section-xxxxs 0;
```

Comment:
xxxxs really?

Reply:
@vincentfretin temp patch because I am currently working on having less variables for font sizes, line height, padding, media queries, etc. those modification couldn't be shipped with this third batch because they wasn't ready. Should be good with the fourth batch.